src/NuGet.Core/NuGet.Frameworks/PublicAPI/net40/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+const NuGet.Frameworks.FrameworkConstants.FrameworkIdentifiers.XamarinMacCatalyst = "Xamarin.MacCatalyst" -> string
```

Comment:
Where is this property declared? Should it be removed as per previous comment?

Reply:
oh no, this was a mistake and I fixed it already but I forgot to push. I've pushed it now. You can ignore this bit :)